pronotepy/clients.py

```diff
@@ -509,24 +510,65 @@ def homework(
                 out.append(hw)
         return out
 
-    def messages(self) -> List[dataClasses.Message]:
+    def get_recipients(self) -> List[dict]:
+        """
+        Get all possible recipients for new discussion
+
+        Returns
+        -------
+        List[dict]
+        """
+        data = {"onglet": {"N": 0, "G": 3}}
+        recipients = self.post("ListeRessourcesPourCommunication", 131, data)[
+            "donneesSec"
+        ]["donnees"]["listeRessourcesPourCommunication"]["V"]
+        data = {"onglet": {"N": 0, "G": 34}}
+        recipients += self.post("ListeRessourcesPourCommunication", 131, data)[
+            "donneesSec"
+        ]["donnees"]["listeRessourcesPourCommunication"]["V"]
+
+        return [r for r in recipients if r.get("avecDiscussion")]
+
+    def new_discussion(self, subjet: str, message: str, recipient: List) -> None:
```

Comment:
`recipient` parameter should be typed better

Reply:
Okay I will do that